wordpress-mobile/WordPress-iOS
# [Jetpack] Shared Defaults Posts

This PR updates the `UserDefaults` instance in `PostListViewController` to use new factory instance and updates the `KeyValueDatabase` conformance.

To test:
1. Enable Shared Defaults by setting the FeatureFlag to true
2. Launch WP iOS & Login (if not already launched and logged in)
3. Go to posts and toggle the grid/list icon from right top.
4. Launch JP iOS.
5. Go to posts and verify if the icon and the list's state matches of the WP app.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
